package.json

```diff
@@ -57,7 +57,6 @@
   "dependencies": {
     "@babel/runtime": "^7.1.2",
     "@segment/react-tiny-virtual-list": "^2.2.1",
-    "@types/react": "^16.9.5",
```

Comment:
Won't this cause TS projects to fail if the type exports are different across versions (since Evergreen relies on them in its own  typedefs)?

Reply:
Since Evergreen now supports react@17 though, maybe we can just update this instead? To avoid react versioning mismatches

Reply:
Added @types/react@^17.0.0 to devDependencies